src/component/GenreButton/GenreButton.test.js

```diff
@@ -0,0 +1,30 @@
+import { render, fireEvent } from "@testing-library/react";
+import GenreButton from "./GenreButton";
+
+describe("GenreButton", () => {
+  it("renders the button with the correct class and label", () => {
+    const genre = "Action";
+    const buttonClass = "red";
+
+    const { getByText, getByRole } = render(
+      <GenreButton genre={genre} buttonClass={buttonClass} onSelect={() => {}} />
```

Comment:
render can be moved to setUp() as suggested above.

Reply:
Fixed as part of 903b467